token/sdk/network/ownership.go

```diff
@@ -13,6 +13,7 @@ import (
 
 type Ownership interface {
```

Comment:
shall we change the name of this interface to Authorization?

Reply:
okay